step.yml

```diff
@@ -227,6 +227,10 @@ inputs:
     opts:
       title: "Force generate universal APK for AABs"
```

Comment:
I'd add the **DEPRECATED** to the title as well. Description is not visible by default.

Reply:
I wanted to do it but didn't find an aesthetically pleasing way. I would say that the least ugly way is to add the `(DEPRECATED)` postfix. Wdyt?